core/controller/src/main/scala/whisk/core/controller/Actions.scala

```diff
@@ -306,7 +302,7 @@ trait WhiskActionsApi
      * - 200 [] or [WhiskAction as JSON]
      * - 500 Internal Server Error
      */
-    override def list(namespace: EntityPath, excludePrivate: Boolean)(implicit transid: TransactionId) = {
+    override def list(user: Identity, namespace: EntityPath, excludePrivate: Boolean)(implicit transid: TransactionId) = {
```

Comment:
it's staged for a subsequent PR in preparation: the user will be required as part of the object access model.